app/tests/doctest/doctest.hpp

```diff
@@ -7103,4 +7103,4 @@ DOCTEST_SUPPRESS_COMMON_WARNINGS_POP
 #ifdef DOCTEST_UNDEF_NOMINMAX
 #undef NOMINMAX
 #undef DOCTEST_UNDEF_NOMINMAX
-#endif // DOCTEST_UNDEF_NOMINMAX
\ No newline at end of file
+#endif // DOCTEST_UNDEF_NOMINMAX
```

Comment:
dont change this file

Reply:
it's a third party lib in case you didnt notice, it should ideally go into third_party directory actually